manifests/profile/neutron/router.pp

```diff
@@ -12,14 +12,17 @@
 
   ### Router service installation
   class { '::neutron::agents::l3':
-    debug                   => $::openstack::config::debug,
-    external_network_bridge => 'brex',
-    enabled                 => true,
+    debug                    => $::openstack::config::debug,
+    external_network_bridge  => 'brex',
+    enabled                  => true,
+    agent_mode               => 'dvr_snat',
+    router_delete_namespaces => 'True',
```

Comment:
Hi @mmailand,

yes, it's plain text string in configure in `l3_agent.ini`,
but it would be better to be configured as Boolean [2]
since `neutron_l3_agent_config` will help us to convert value into plain text string[1],

[1] https://github.com/openstack/puppet-neutron/blob/master/lib/puppet/type/neutron_l3_agent_config.rb#L14
[2] https://github.com/openstack/puppet-neutron/blob/master/manifests/agents/l3.pp#L75